### PR TITLE
Add bucket fields to oracle source type

### PR DESCRIPTION
**Pull request by lpichler**
https://issues.redhat.com/browse/RHCLOUD-23393

- This adds 3 bucket fields.
- changes username to default value - https://issues.redhat.com/browse/RHCLOUD-20351

### CODE REVIEW

File: dao/seeds/source_types.yml [hunk 1]
```diff
@@ -302,13 +302,28 @@ oracle-cloud-infrastructure:
...
           - component: text-field
-            isRequired: true
-            label: Oracle Cloud ID (OCID)
+            hideField: true
+            initializeOnMount: true
+            initialValue: ocid.uuid
             name: authentication.username
+          - component: text-field
+            name: application.extra.bucket
+            label: Bucket Namespace
```
Comment (lindgrenj6) [inline dao/seeds/source_types.yml]: Should this be `Bucket Name`

File: dao/seeds/source_types.yml [hunk 2]
```diff
@@ -302,13 +302,28 @@ oracle-cloud-infrastructure:
...
+            isRequired: true
+            validate:
+              - type: required
+          - component: text-field
+            name: application.extra.bucket_namespace
+            label: Bucket Name
```
Comment (lindgrenj6) [inline dao/seeds/source_types.yml]: then this one Namespace

File: dao/seeds/source_types.yml [hunk 3]
```diff
@@ -302,13 +302,28 @@ oracle-cloud-infrastructure:
...
+            isRequired: true
+            validate:
+              - type: required
+          - component: text-field
+            name: application.extra.bucket_region
```
Comment (lindgrenj6) [inline dao/seeds/source_types.yml:322]: then this one Region?

File: dao/seeds/source_types.yml [hunk 4]
```diff
@@ -302,13 +302,28 @@ oracle-cloud-infrastructure:
...
+            label: Bucket Name
+            isRequired: true
             validate:
               - type: required
-              - type: pattern
-                pattern: "^ocid1\\..*"
```
Comment (lindgrenj6) [inline dao/seeds/source_types.yml:311]: Did we validate that the ID isn't like this pattern? It's ok if it isn't - just want to make sure.

Comment (lpichler) [reply dao/seeds/source_types.yml:311]: We did but now we don't need ID from user. Cost needs here default value "ocid.uuid" - see more here https://issues.redhat.com/browse/RHCLOUD-20351?focusedCommentId=21411257&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21411257

File: dao/seeds/source_types.yml (hunk 1 above)
Comment (lpichler) [reply dao/seeds/source_types.yml]: Yes I fixed all. I probably got lost in yaml syntax 🤦

File: dao/seeds/source_types.yml (hunk 4 above)
Comment (lindgrenj6) [reply dao/seeds/source_types.yml:311]: sweeeet sounds good!

**Review by lindgrenj6 — APPROVED** [review]

**Pull request merged**